public/achievementInfo.php

```diff
@@ -146,6 +146,24 @@ function updateAchievementTypeFlag(typeFlag) {
     }
     </script>
 <?php endif ?>
+
+<?php if ($achievedLocal): ?>
```

Comment:
personally i'd opt for default form post request behaviour instead of emulating it via javascript.
i think adjusting the input name should be enough?

Reply:
reset-achievements.php returns JSON, not a web page or redirect.